src/app/facilities/facilities.component.ts

```diff
@@ -0,0 +1,36 @@
+import {AfterViewInit, Component, OnInit} from '@angular/core';
+import {FacilitiesDescriptions} from "./utilities/faciilities-descriptions";
+import {animate, animation, style, transition, trigger} from "@angular/animations";
+import {fadeSlideInOutAnimation} from "../shared/animations";
+
+@Component({
+  selector: 'app-facilities',
+  templateUrl: './facilities.component.html',
+  styleUrls: ['./facilities.component.scss'],
+  animations: [fadeSlideInOutAnimation]
+})
+export class FacilitiesComponent implements OnInit{
+    public descriptionToDisplay?: string;
+
+    private observer = new IntersectionObserver((entries) => {
+        entries.forEach((entry) => {
+            if (entry.isIntersecting) {
+                entry.target.classList.add('visible');
+            }
+        });
+    }, { threshold: 0.2 });
+
+    ngOnInit(){
+        this.descriptionToDisplay = 'first';
```

Comment:
Nie podoba mi się to że korzystamy tutaj ze stringa zwykłego, zamieniłbym te stringi na liczby jak już

Reply:
Teraz spojrzałem jeszcze jak z tego korzystamy i liczby nie będą najlepszym rozwiązaniem. Zrobiłbym to tak że w tym FacilitiesDescriptions zamiast posiadać pól first, second itd. dałbym pola od tytułów zdjęć, co prawda nie jest to najlepsze rozwiązanie, bo w przypadku jakiś zmian będzie wszystko trzeba zmienić, ale aktualnie nic lepszego nie wymyśliłem

Reply:
Teraz spojrzałem jeszcze jak z tego korzystamy i liczby nie będą najlepszym rozwiązaniem. Zrobiłbym to tak że w tym FacilitiesDescriptions zamiast posiadać pól first, second itd. dałbym pola od tytułów zdjęć, co prawda nie jest to najlepsze rozwiązanie, bo w przypadku jakiś zmian będzie wszystko trzeba zmienić, ale aktualnie nic lepszego nie wymyśliłem